### PR TITLE
Add custom container

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: custom/custom/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        dbsession = request.environ.get("app.dbsession")
+        if dbsession is None:
+            # request.tm is the transaction manager used by pyramid_tm
+            dbsession = get_tm_session(session_factory, request.tm, request=request)
+        return dbsession
+
+    config.add_request_method(dbsession, reify=True)
+
+
+class Feedback(Base):
```
Comment (arnaud-morvan) [inline custom/custom/models/__init__.py]: I would create a dedicated module for the models, for example `feedback.py`

File: custom/README.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    python3 -m venv env
+
+- Upgrade packaging tools, if necessary.
+
+    env/bin/pip install --upgrade pip setuptools
+
+- Install the project in editable mode with its testing requirements.
+
+    env/bin/pip install -e ".[testing]"
```
Comment (arnaud-morvan) [inline custom/README.txt]: Not sure this is up to date with Docker and pipenv.

**Review by arnaud-morvan — COMMENTED** [review]
Note that would name the folder feedback instead of custom.
I would also like to test it if I found enough time.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "custom",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "lit": "2.0.0-rc.3",
```
Comment (llienher) [inline package.json]: We could use a relased version, even if I know there is no consequence to keep it. :-)

**Review by llienher — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (sbrunner) [reply package.json]: +1 :-)

**Pull request merged**